src/main/scala/org/ergoplatform/ErgoLikeContext.scala

```diff
@@ -162,3 +162,9 @@ case object Self extends NotReadyValueBox {
   override val opCode: OpCode = OpCodes.SelfCode
   def opType = SFunc(SContext, SBox)
 }
+
+case object Context extends NotReadyValue[SContext.type] {
+  override val opCode: OpCode = OpCodes.ContextCode
+  override def tpe: SContext.type = SContext
+  override def opType: SFunc = SFunc(SUnit, SContext)
+}
```

Comment:
@greenhat We don't need such node in the tree, because any script if `f: Context => SigmaProp`, so to context is actually argument of the function.

Reply:
@aslesarenko If we remove `Context` case object we need to translate every context method call into an IR node in the binder. This means no `MethodCall`s. Or am I missing something?

Reply:
@greenhat We will still keep existing context operations as shortcuts for method of Context.
But we don't need new shortcuts.
And remember is the graph IR, `HEIGHT` becomes `ctx.HEIGHT` where ctx is lambda var.

Reply:
@aslesarenko Got it. Removed.